cli/crates/federated-dev/src/dev/gateway_nanny.rs

```diff
@@ -91,7 +92,7 @@ pub(super) async fn new_gateway(config: Option<engine_v2::VersionedConfig>) -> O
         entity_cache: InMemoryEntityCache::default(),
     };
 
-    let schema = config.try_into().ok()?;
+    let schema = (config, graph).try_into().ok()?;
```

Comment:
As mentioned later, think it's more harmful to have this API than the small gains in a barely used cli feature (only meant for development) this adds.

Reply:
done